pyfar/classes/coordinates.py

```diff
@@ -41,6 +41,10 @@
 systems (`spherical_colatitude` and `spherical_elevation`). The table below
 lists all coordinates.
 
+.. note::
+    All coordinates are returned as copies of the internal data. This means
+    that changing values within slices (e.g. ``coordinates.x[0] = 0``)
+    has no effect on the :py:class:`~pyfar.Coordinates` object's data.
```

Comment:
Maybe it is clearer if we give an example of how to do this:
```suggestion
    that for example ``coordinates.x[0] = 0`` does not change
    ``coordinates.x``. This can be done using
    
    .. code-block:: python

       new_x = coordinates.x
       new_x[0] = 0
       coordinates.x = new_x
```